src/actions/secrets.js

```diff
@@ -78,7 +78,10 @@ export function deleteSecret(secrets) {
     which is the only way to remove secrets otherwise using the patch API */
 
     // For each namespace there are secrets in
-    namespacesToSecretsMap.forEach(async function(value, namespace) {
+    namespacesToSecretsMap.forEach(async function handleSecretUnpatching(
+      value,
+      namespace
+    ) {
```

Comment:
Was this an intentional change to have in this PR? Seems unrelated to the rest of the PR (not that this is a bad code change).

Reply:
Renamed a variable, ran the lint fix thus it's split over the new lines